data-plane/src/health/initialized.rs

```diff
@@ -1,15 +0,0 @@
-pub trait InitializedHealthcheck {
-    type Error: std::error::Error;
-    fn is_initialized(&self) -> Result<bool, Self::Error>;
-}
-
-pub struct EnclaveEnvInitialized;
-
-impl InitializedHealthcheck for EnclaveEnvInitialized {
```

Comment:
this seems unnecessary so moved into the agent. Also, we weren't handling the case where the customer-env file hadn't been written yet. It's a bit weird this wasn't causing our tests to fail.